public/plugins/query_editors/sql/sql_raw_query_editor.tsx

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { EuiSpacer } from '@elastic/eui';
+import { MonacoRawQueryEditor } from '../monaco_raw_query_editor';
+import { QueryHeader } from '../query_header';
+
+export const SqlRawQueryEditor = (props) => {
```

Comment:
np. add type for props?

Reply:
Will add for sure, now it's not yet finalized what are the props it needs